pyaedt/icepak.py

```diff
@@ -745,7 +752,7 @@ def create_network_block(
         """Create a network block.
 
         .. deprecated:: 0.6.27
-            This method will be replaced by `create_two_resistor_network_block`.
+            This method will be replaced by ``create_two_resistor_network_block``.
```

Comment:
```suggestion
            This method is to be replaced by the ``create_two_resistor_network_block`()` method.
```
Use present tense. Also possible use a link to the replacement method.